### PR TITLE
Replica branch

### PR DESCRIPTION
**Pull request by tahiry-dev**
- I used the Toggle Pesticide extension to understand better the structure of the HTML.
- Write HTML Tags for all the sections and versions (large, medium, and small view).
- Style CSS from the mobile point of view from the navigation to the footer
- Adjust repetitive CSS
- Fix all the coding syntax according to "Webhint" and "Stylehint"
- Write readme file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,11 +5,1019 @@
...
+                    <i class="fas fa-bars fa-lg"></i>
+                </div>
+            </div>
+        </div>
+        <nav class="follow-up-link">
+            <a class="no-decoration black-color" href="#">Stay Home. Go global. Build your network at
+                TNW2020 Online
+                <i class="fas fa-arrow-right"></i></a>
+        </nav>
+    </div>
```
Comment (arslanbisharat) [inline index.html]: Please try to use header

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
### Request changes 😳
I am very happy to see this project and I can see you worked hard on this project but I am sorry to inform you that I cannot approve this project because it is not completed yet.😟 You need to work more on it. 
_It's okay. We are here to learn. Don't get discouraged. Work even harder. You are almost there.😄_
**Please have a look at the suggestions to get more understanding.**


- [ ] TNW logo at the top bar is not a word. Please make it like the original one. 
chrome-capture (2)

- [ ] On small and medium-size screens the logo is not aligned as in the original page. 
chrome-capture (3)


- [ ] TNW logo at the bottom is not correctly aligned and its size is bigger than the original one. Please wok on it.



chrome-capture
- [ ] Remove conflict in files
- [ ] Increase the space between links

chrome-capture (1)


- [ ] Optional Please make it more descriptive
chrome-capture (5)
. 

**Please make the changes and request re-review for the given project**
Stay safe and give me a smile 😇

### Happy Coding
------------------------------------------------------------------------------------------------
`**Regards,`
Arslan Bisharat
slack: arslanbisharat**

**Review by teekaytech — APPROVED** [review]
# Status : Approved :green_circle: 

### You've done a nice job implementing the changes raised by the previous TSE. Therefore, this project is approved. You can now merge to master! :tada: 

**Wish you the best in your next project :muscle:**
**See you at the top :100:**

**Happy coding** :man_technologist:

**Comment by tahiry-dev** [comment]
Thank you sir!

Le lun. 21 sept. 2020 à 19:41, Taofeek Olalere <notifications@github.com> a
écrit :

**Pull request merged**